backend/internal/server/apis/schedule/games/manual/games.go

```diff
@@ -0,0 +1,193 @@
+package manual
+
+import (
+	"github.com/gofiber/fiber/v2"
+	"github.com/jak103/powerplay/internal/db"
+	"github.com/jak103/powerplay/internal/models"
+	"github.com/jak103/powerplay/internal/server/apis"
+	"github.com/jak103/powerplay/internal/server/apis/schedule/internal/mapping"
+	"github.com/jak103/powerplay/internal/server/apis/schedule/internal/structures"
+	"github.com/jak103/powerplay/internal/server/services/auth"
+	"github.com/jak103/powerplay/internal/utils/log"
+	"github.com/jak103/powerplay/internal/utils/responder"
+)
+
+func init() {
+	apis.RegisterHandler(fiber.MethodPost, "/schedule/manual/game", auth.Authenticated, handleCreateGame)
+	apis.RegisterHandler(fiber.MethodPost, "/schedule/manual/games", auth.Authenticated, handleCreateGames)
+	apis.RegisterHandler(fiber.MethodGet, "/schedule/manual/games/:id", auth.Authenticated, handleGetGame)
+	apis.RegisterHandler(fiber.MethodGet, "/schedule/manual/games", auth.Authenticated, handleGetGames)
+	apis.RegisterHandler(fiber.MethodPut, "/schedule/manual/games/:id", auth.Authenticated, handleUpdateGame)
+	apis.RegisterHandler(fiber.MethodPut, "/schedule/manual/games", auth.Authenticated, handleUpdateGames)
+	apis.RegisterHandler(fiber.MethodDelete, "/schedule/manual/games/:id", auth.Authenticated, handleDeleteGame)
+	apis.RegisterHandler(fiber.MethodDelete, "/schedule/manual/games", auth.Authenticated, handleDeleteGames)
+}
+
+// basic CRUD operations
+// - the caller is allowed to fine tune the schedule by updating the games
+
+func handleCreateGame(c *fiber.Ctx) error {
+	type Dto struct {
+		Game structures.Game `json:"game"`
+	}
+	var dto Dto
+	err := c.BodyParser(&dto)
+	if err != nil {
+		log.Error("Failed to parse request body")
```

Comment:
Should we make logs like this more specific by saying something like "Failed to parse request body for handleCreateGame"? or will the error also have a stack trace?

Reply:
There will be log of what was called before this